From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <michael.i.doherty@intel.com>
Date: Tue, 2 May 2017 18:13:05 +0100
Subject: [PATCH] pam: Allow gnome-initial-setup to operate in
 gdm-launch-environment

This change is required so that both GDM shell and GNOME Initial Setup
can operate fully within the GDM launch environment, i.e for the initial
setup experience.

Signed-off-by: Ikey Doherty <michael.i.doherty@intel.com>
---
 data/pam-lfs/gdm-launch-environment.pam | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/data/pam-lfs/gdm-launch-environment.pam b/data/pam-lfs/gdm-launch-environment.pam
index 86f553c..9645389 100644
--- a/data/pam-lfs/gdm-launch-environment.pam
+++ b/data/pam-lfs/gdm-launch-environment.pam
@@ -1,15 +1,15 @@
 # Begin /etc/pam.d/gdm-launch-environment
 
-auth     required       pam_succeed_if.so audit quiet_success user = gdm
+auth     required       pam_succeed_if.so audit quiet_success user in gdm:gnome-initial-setup
 auth     required       pam_env.so
 auth     optional       pam_permit.so
 
-account  required       pam_succeed_if.so audit quiet_success user = gdm
+account  required       pam_succeed_if.so audit quiet_success user in gdm:gnome-initial-setup
 account  include        common-account
 
 password required       pam_deny.so
 
-session  required       pam_succeed_if.so audit quiet_success user = gdm
+session  required       pam_succeed_if.so audit quiet_success user in gdm:gnome-initial-setup
 -session optional       pam_systemd.so
 session  optional       pam_keyinit.so force revoke
 session  optional       pam_permit.so
